src/displayapp/screens/Dice.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "displayapp/screens/Screen.h"
+#include "displayapp/widgets/Counter.h"
+#include "systemtask/SystemTask.h"
+#include <components/motion/MotionController.h>
+#include <lvgl/lvgl.h>
+#include <random>
+#include <array>
```

Comment:
inside a group of includes (like std-lib includes) try to order them alphabetically (if possible)

```suggestion
#include <array>
#include <random>
```

Reply:
fixed in 39235224f476390ad2e5ffc3418ca96885a491d1